lib/api.c

```diff
@@ -212,9 +215,12 @@ int request_table_addr(ryzen_access ry)
 
 	resp = smu_service_req(ry->psmu, get_table_addr_msg, &args);
 
-	if (ry->family == FAM_REMBRANDT) {
+	switch (ry->family)
+	{
+	case FAM_REMBRANDT:
+	case FAM_PHEONIX:
 		ry->table_addr = (uint64_t) args.arg1 << 32 | args.arg0;
-	} else {
+	default:
 		ry->table_addr = args.arg0;
```

Comment:
The problem is here. I changed the default to this `ry->table_addr = (uint64_t) args.arg1 << 32 | args.arg0;` and it works.

Reply:
Yes, thanks for that. I just forgot the break statement.